_data/pt-br/h.yml

```diff
@@ -1,3 +1,12 @@
+- title: Haskell
+  tags:
+    - Backend
+  id: haskell
+  description: "Haskell é uma linguagem de programação funcional, com tipagem forte e estática, 
```

Comment:
Talvez essa explicação fique difícil para quem não é da área de programação. Termos como 'tipagem forte', 'avaliação preguiçosa' e 'funcional' podem não ser claros para todas as pessoas. Que tal reescrever de forma mais acessível, explicando esses conceitos com exemplos simples ou comparações? Aqui uma sugestão:

Haskell é uma linguagem de programação usada para criar softwares. Ela segue um estilo diferente chamado 'funcional', onde as instruções são escritas como se fossem fórmulas matemáticas. Em Haskell, o computador só calcula o que for realmente necessário, o que pode tornar os programas mais rápidos e eficientes. Ela também ajuda a evitar muitos erros comuns, por isso é muito usada em pesquisas e projetos que precisam ser muito confiáveis.